clang/include/clang/AST/DeclarationName.h

```diff
@@ -769,17 +774,12 @@ class DeclarationNameLoc {
     DNL.setNamedTypeLoc(TInfo);
     return DNL;
   }
-
-  /// Construct location information for a non-literal C++ operator.
-  static DeclarationNameLoc makeCXXOperatorNameLoc(SourceLocation BeginLoc,
-                                                   SourceLocation EndLoc) {
-    return makeCXXOperatorNameLoc(SourceRange(BeginLoc, EndLoc));
-  }
-
+  
   /// Construct location information for a non-literal C++ operator.
-  static DeclarationNameLoc makeCXXOperatorNameLoc(SourceRange Range) {
+  static DeclarationNameLoc
+  makeCXXOperatorNameLoc(CXXOperatorSourceInfo *OInfo) {
```

Comment:
This is a weird function to exist... am I correct in that it is only called 1x?  I find myself thinkin perhaps we should just inline it manually.